wkcuber/cubing.py

```diff
@@ -64,9 +64,9 @@ def create_parser():
 
     return parser
 
-def read_image_file(file_name):
+def read_image_file(file_name, dtype):
```

Comment:
Where is this function used? Did you check that it's never called without the `dtype` parameter?

Reply:
This function is called from `cubing_job` only